bpaotu/bpaotu/frontend/package.json

```diff
@@ -62,6 +62,7 @@
     "react-leaflet": "^1.9.1",
     "react-leaflet-fullscreen": "^0.0.6",
     "react-leaflet-markercluster": "^1.1.8",
+    "react-loading-spin": "^1.0.8",
```

Comment:
Did you add this with `yarn` or `npm`?

Because, the `yarn.lock` file below doesn't seem to have the package...

Reply:
yarn add react-loading-spin --dev

Reply:
done